core/primitives/common.hpp

```diff
@@ -0,0 +1,13 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef KAGOME_CORE_PRIMITIVES_COMMON_HPP_
+#define KAGOME_CORE_PRIMITIVES_COMMON_HPP_
```

Comment:
```suggestion
#define KAGOME_CORE_PRIMITIVES_COMMON_HPP
```
I think, your default template adds an underscore, which is not needed. Please, check

Reply:
Mine contains underscore as well, I remove it every time(